@shared/api/types.ts

```diff
@@ -245,10 +246,9 @@ export interface Balance {
   blockaidData: BlockAidScanAssetResult;
 }
 
-export interface BlockAidScanAssetResult {
-  result_type: "Benign" | "Warning" | "Malicious" | "Spam";
-  features: { description: string }[];
-}
+export type BlockAidScanAssetResult = Blockaid.TokenScanResponse;
+export type BlockAidScanSiteResult = Blockaid.SiteScanResponse;
+export type BlockAidScanTxResult = Blockaid.StellarTransactionScanResponse;
```

Comment:
I'm exporting all the types from this file so we only have to worry about keeping "@blockaid/client" updated in one subrepo